frame/elections-phragmen/src/lib.rs

```diff
@@ -766,14 +770,16 @@ impl<T: Trait> Module<T> {
 			// save the members, sorted based on account id.
 			new_members.sort_by(|i, j| i.0.cmp(&j.0));
 
-			let mut prime_votes: Vec<_> = new_members.iter().map(|c| (&c.0, BalanceOf::<T>::zero())).collect();
+			let mut prime_votes: Vec<_> = new_members.iter().map(|c| (&c.0, VoteWeight::zero())).collect();
 			for (_, stake, targets) in voters_and_votes.into_iter() {
 				for (votes, who) in targets.iter()
 					.enumerate()
 					.map(|(votes, who)| ((MAXIMUM_VOTE - votes) as u32, who))
 				{
 					if let Ok(i) = prime_votes.binary_search_by_key(&who, |k| k.0) {
-						prime_votes[i].1 += stake * votes.into();
+						// let () = stake;
+						// let () = votes;
+						prime_votes[i].1 += stake * votes as VoteWeight;
```

Comment:
can you explain to me why is prime member computed this way? why does it multiplies with the vote_index in target?

Reply:
https://github.com/paritytech/substrate/issues/5290

Reply:
(maybe worth having it quickly documented, but it is indeed not the scope of the PR)